scripts/simulate.py

```diff
@@ -0,0 +1,192 @@
+"""
+CLI for hera_sim
+"""
+import argparse
+import copy
+import os
+import sys
+import warnings
+import yaml
+
+import hera_sim
+from hera_sim import cli_utils
+from pyuvsim.simsetup import _parse_layout_csv
+from astropy.coordinates import Angle
+from astropy import units
+try:
+    import bda
+    from bda import bda_tools
+except ImportError:
+    warnings.warn("bda failed to import. Baseline-dependent averaging not available.")
+    bda = None
+
+parser = argparse.ArgumentParser(
+    description="Run a hera_sim-managed simulation from the command line."
+)
+parser.add_argument("config", type=str, help="Path to configuration file.")
+parser.add_argument(
+    "-o",
+    "--outfile",
+    type=str,
+    default=None,
+    help="Where to save simulated data. Overrides outfile specified in config."
+)
+parser.add_argument(
+    "-v",
+    "--verbose",
+    default=False,
+    action="store_true",
+    help="Print progress updates."
+)
+parser.add_argument(
+    "-sa",
+    "--save_all",
+    default=False,
+    action="store_true",
+    help="Save each simulation component."
+)
+parser.add_argument(
+    "--clobber",
+    default=None,  # To allow the user to specify clobber in the config file.
+    action="store_true",
+    help="Overwrite existing files in case of name conflicts."
+)
+args = parser.parse_args()
+
+if args.verbose:
```

Comment:
Shouldn't everything from here down be in a `if __name__ == "__main__"`?

Reply:
I think that is best practice, yes. I'll make the update.